bin/generate-post-meta-image

```diff
@@ -1,34 +1,68 @@
 #!/bin/sh -e
 
-text="$1"
-bg="$2"
-out=$3
+tmp=$(mktemp -d)
+bg="#2421AB"
 res=2160x1080
 
-tmp=$(mktemp -d)
+while [[ $# -gt 0 ]] && [[ "$1" == "--"* ]]; do
```

Comment:
Either I'm missing something or these conditions do not allow for short flags. Does "-r" match here for instance?

Reply:
true. missed that case (this was originally copy/pasted from another script

but in hindsight, and answering your other comment, I guess I can just remove support for the shorthand syntax

Reply:
fixed